### DISCUSSION TITLE
Is or will FerretDB be multithreaded?

### DISCUSSION THREAD

**Discussion by ghost** [FerretDB/FerretDB#134] [General]
Hello,

Does FerretDB currently support threads? Is there any plan to do so if not?

Thanks, look forward to following FDB's development.

**Comment by AlekSi** [comment]
Each client connection uses a [goroutine](https://go.dev/tour/concurrency/1); several other goroutines are used for other tasks. So yes, FerretDB is quite multi-threaded.
